src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -143,6 +143,11 @@ $left-sidebar-z-index: 40 !default;
     .Body.is-responsive-web {
         .bdl-CollapsibleSidebar-wrapper {
             width: 0;
+            margin-top: 60px;
```

Comment:
Apparently, if the number of pixels for any spacing related style is divisible by four, we have to use `bdlGridUnit`---in this case, `bdlGridUnit * 15`

Reply:
Decided this isn't necessary